presto-product-tests/bin/product-tests-suite-1.sh

```diff
@@ -10,7 +10,7 @@ suite_exit_code=0
 presto-product-tests-launcher/bin/run-launcher test run \
     --environment multinode \
     -- \
-    -x quarantine,big_query,storage_formats,profile_specific_tests,tpcds,cassandra,mysql,postgresql,kafka,hive_compression,"${DISTRO_SKIP_GROUP}" \
```

Comment:
I would keep `quarantine`

Reply:
i thought about that. Note that quarantine was plugged in suite-1n only, and not in the others suites.
So if you wanted to quarantine some other test, you still need to do a modification of a suite.

Let's drop it now, and readd (where necessary) later, if needed.